keptn-jenkins-library/5.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,18 @@
+version: '5.0'
+name: keptn-jenkins-library
+displayName: keptn-jenkins-library
+createdAt: '2021-10-10T07:27:14Z'
+description: Jenkins shared library for integrating Keptn Use Cases with your Jenkins
+  Pipelines
+digest: '2021-11-11T10:11:44Z'
+license: Apache-2.0
+homeURL: https://keptn.sh/docs/integrations/
+keywords:
+- keptn
```

Comment:
`sandbox` and `testing` need to be added to make the search work. I'll add this to the python file so this is done automatically in the future.

Reply:
Done